server/views/autherror.njk

```diff
@@ -2,12 +2,25 @@
 
 {% set pageId = 'auth-error' %}
 
+{% block beforeContent %}
```

Comment:
As well as updating the page content, the breadcrumbs were also missing from the authorisation error, 404, and general error pages.